content/docs/user-guide/gems/reference/aws/aws-gamelift/gem-setup.md

```diff
@@ -21,29 +21,29 @@ The lifetime of a session involves the following phases:
    4. Destroy the session when the condition is met. For example, when the last player leaves the session. 
```

Comment:
```
description: "Learn how to set up the AWS GameLift Gem in O3DE"
toc: true
weight: 100
---
This topic teaches you how to set up the AWS GameLift Gem in Open 3D Engine (O3DE) and the AWS GameLift service in your project.
```
I see that we need a couple of changes in the metadata and intro sentence above (not in your changes). Please make these fixes:
* For the description: Drop the quotation marks, use the full name of O3DE, and add a period:
>Learn how to set up the AWS GameLift Gem in Open 3D Engine (O3DE).
* In the intro:
  * Add **bold** formatting to the Gem name first reference.
  * In the intro, change "the AWS GameLift service" to "Amazon GameLift". (Incorrect naming for the AWS service.)